addons/mail/static/src/core/common/chat_window.js

```diff
@@ -132,6 +132,12 @@ export class ChatWindow extends Component {
                 }
                 break;
             }
+            case "k":
+                if(ev.ctrlKey) {
```

Comment:
```suggestion
                if (ev.ctrlKey) {
```
Also, I think we should have a small unit test for this one